examples/lband_decode/lband_decode.cc

```diff
@@ -22,12 +26,21 @@ void OnFEMessage(const MessageHeader& header, const void* data) {
   if (header.message_type == LBandFrameMessage::MESSAGE_TYPE) {
     auto frame = reinterpret_cast<const LBandFrameMessage*>(data);
     auto lband_data =
-        static_cast<const char*>(data) + sizeof(LBandFrameMessage);
+        static_cast<const uint8_t*>(data) + sizeof(LBandFrameMessage);
     printf("Decoded %u L-band bytes.\n", frame->user_data_size_bytes);
-    out_stream.write(lband_data, frame->user_data_size_bytes);
+    rtcm_framer.OnData(lband_data, frame->user_data_size_bytes);
   }
 }
 
+// This is the callback for handling decoded RTCM messages.
+/******************************************************************************/
+void OnRTCMMessage(uint16_t message_type, const void* data, size_t data_len) {
+  // Don't warn unused.
+  (void)data;
```

Comment:
I often find it's a little simpler to just comment out the argument name above like `const void* /*data*/` instead of doing this.